settings.gradle.kts

```diff
@@ -3,31 +3,21 @@
  */
 
 pluginManagement {
+    includeBuild("gradle-settings-conventions")
+
     repositories {
         mavenCentral()
         google()
         gradlePluginPortal()
     }
 }
 
-rootProject.name = "ktor"
-
-val CACHE_USER = System.getenv("GRADLE_CACHE_USER")
-
-if (CACHE_USER != null) {
-    val CACHE_PASSWORD = System.getenv("GRADLE_CACHE_PASSWORD")
-    buildCache {
-        remote(HttpBuildCache::class) {
-            isPush = true
-            setUrl("https://ktor-gradle-cache.teamcity.com/cache/")
```

Comment:
Can't find this link in the new configuration. Is it hidden on the Develocy server?